### PR TITLE
fix(cache): Cache deployer - Using the same kubectl version as the server

### PR DESCRIPTION
**Pull request by Ark-kun**
Fixes https://github.com/kubeflow/pipelines/issues/4505

### CODE REVIEW

File: backend/src/cache/deployer/deploy-cache-service.sh [hunk 1]
```diff
@@ -26,6 +26,18 @@ NAMESPACE=${NAMESPACE_TO_WATCH:-kubeflow}
 MUTATING_WEBHOOK_CONFIGURATION_NAME="cache-webhook-${NAMESPACE}"
 WEBHOOK_SECRET_NAME=webhook-server-tls
 
+# Getting correct kubectl. Kubernetes only supports kubectl versions within +/-1 minor version.
+# kubectl has some resource version information hardcoded, so using too old kubectl can lead to errors
+mkdir -p "$HOME/bin"
+export PATH="$PATH:$HOME/bin"
```
Comment (Bobgy) [inline backend/src/cache/deployer/deploy-cache-service.sh:32]: You are putting `HOME` after `PATH`, so I think system will still use the default kubectl.
Did I miss anything?

Comment (Ark-kun) [reply backend/src/cache/deployer/deploy-cache-service.sh]: Thank you for noticing. You're correct.
I guess it was my "muscle memory" regarding adding things to PATH.
Fixed.

**Comment by Bobgy** [comment]
/retest
/lgtm
/approve
Thanks!
looks like a temporary failure

**Comment by Bobgy** [comment]
/lgtm
Thanks for the fix!

**Pull request merged**